torchrl/objectives/dreamer.py

```diff
@@ -160,6 +199,12 @@ def forward(self, tensordict: TensorDict) -> torch.Tensor:
             tensordict.detach(),
         )
 
+    @staticmethod
+    def normal_log_probability(x, mean, std):
+        return (
+            -0.5 * ((x.to(mean.dtype) - mean) / std).pow(2) - std.log()
+        )  # - 0.5 * math.log(2 * math.pi)
+
```

Comment:
here's the equation of the normal log prob if you're interested. It's a distance loss as you can see